src/content/docs/journey-app-slow/root-causes.mdx

```diff
@@ -0,0 +1,92 @@
+
+Before you can improve your site's performance, you need the ability to see what's going on. New Relic monitoring always starts with an **integration**, or a piece of code that collects data from a relevant technology and reports it to New Relic. To monitor your application's performance, you will use an agent created specifically for your app's language.
+
+Click a logo to install an agent. This will send you to the New Relic platform where you will be guided through installing and configuring the agent.
+
+<TechTileGrid>
+  
+  <TechTile
+    name="Go agent"
+    icon="logo-go"
+    to="https://one.newrelic.com/nr1-core?state=985d4005-ba90-a8c7-1da1-2af34539b03b"
+  />
+
+  <TechTile
+    name="Java agent"
+    icon="logo-java"
+    to="https://one.newrelic.com/nr1-core?state=80d18bcb-4919-1fcb-2b77-9406838eb916"
+  />
+
+  <TechTile
+    name=".NET agent"
+    icon="logo-dotnet"
+    to="https://one.newrelic.com/nr1-core?state=30e93090-6dfa-6b70-8e75-472f54414355"
+  />
+
+  <TechTile
+    name="Node.js agent"
+    icon="logo-nodejs"
+    to="https://one.newrelic.com/marketplace/install-data-source?state=be2e62fa-cc3b-c428-27c4-8d662c9e80a1"
+  />
+
+  <TechTile
+    name="PHP agent"
+    icon="logo-php"
+    to="https://one.newrelic.com/nr1-core?state=aa633b41-72d4-009c-3abf-55dcf64894fe"
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+
+</TechTileGrid>
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core)** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
+
+## Identify optimization opportunities [#identify]
+
+APM uses dozens of dashboards and charts. Let's talk about what you should focus on first:
+
+<SideBySide>
+    <Side>
+Look at the **Web transactions time** chart. This chart displays the average response time of certain metrics within your app. As your chart populates with data, take note of strange spikes in any line or segment. At the same time, take note of any segments or lines that consistantly take a large amount of time. 
+    </Side>
+    <Side>
+<img
+  title="Default web transactions chart"
+  alt="Default web transactions chart"
+  src={journeyWebTrans}
+/>
+    </Side>
+</SideBySide>
+
+
+The tiles below explain what each segment represents and will take you to instructions on how to optimize them if you see any spikes or high load:
+
+<DocTiles>
+  <DocTile title='Blue line: application code' path="/docs/journey-app-slow/problematic-transactions" >The blue line represents application code, specifically time spent in your application related to the code that isn't external or database calls.</DocTile>
```

Comment:
What about something like this:

Tile Title: `If it's the blue line`
Tile Body: `The blue line is **application code**, specifically time spent in your application on code that isn't external or database calls. To troubleshoot this, click here for tutorial step 2`

and then subsequent ones can do similar.